setlexsem/utils.py

```diff
@@ -53,6 +53,17 @@ def get_study_paths(
     return path_study_folder, path_results
 
 
+def get_prompt_file_path(sampler_hp, prompt_hp, random_seed):
+    sampler_name = make_sampler_name_from_hps(sampler_hp)
```

Comment:
Add a docstring to explain the use of `random_seed`.

Reply:
Will do